src/Fiona/run_classroom_seating.py

```diff
@@ -4,82 +4,71 @@
 import matplotlib.pyplot as plt
 import networkx as nx
 from matplotlib import animation
+import pickle
+import sys
+from matplotlib.text import OffsetFrom
 
-"""
-Run simulations of the ClassroomModel and visualize the seating process
-
-Usage: python3 run_classroom_seating.py
-"""
-
+MODEL_DATA_PATH = "_model_data.json"
+NUM_ITERATIONS = 300
 
 """
-Determine the seating distribution of a given model instance
-
-Args:
-    model: the classroom model to analyse
+Run simulations of the ClassroomModel and visualize the seating process
 
-Returns:
-    seating_distr: 2D array representing the classroom with aisles, entrances, seats and students
+Usage:
+    python3 run_classroom_seating.py generate
+    python3 run_classroom_seating.py animate
 """
-def get_seating_distribution(model):
-    seating_distr = np.zeros((model.grid.width, model.grid.height))
-    for cell in model.grid.coord_iter():
-        content, x, y = cell
-        for agent in content:
-            if type(agent) is Seat:
-                if agent.student is None:
-                    # seat is available
-                    seating_distr[x,y] = -5
-                else:
-                    # seat is occupied. Determine happiness of the student
-                    seating_distr[x,y] = 5 + agent.get_total_utility(agent.student)
-
-                    #seating_distr[x,y] = 2
-
-    for pos in model.classroom.entrances:
-        seating_distr[pos[0],pos[1]] = -15
 
-    return seating_distr
 
 """
-Determine the "stand-up-cost" for all seats
+Determine relevant properties of the current model state and create an image representation
 
 Args:
     model: the classroom model to analyse
     utilities: list of booleans [location, sociability, friendship] specifying which utilities are used in the model
 
 Returns:
-    blocking: 2D array representing the classroom.
+    image: 2D array representing the classroom.
                 - The level of seat blocking is represented by decreasing negative values (the smaller the costlier to reach the seat)
                 - The level of "happiness" of each student is represented by positive values (the larger the happier)
+    info: 3D array comprising the 4 matrices 'seat_utilities', 'student_IDs', 'student_sociabilities' and 'initial_happiness'
 """
-def get_seat_blocking(model, utilities):
```

Comment:
I tried running via `python3 run_classroom_seating.py generate` and got the following:

```
  File "run_classroom_seating.py", line 119, in <module>
    image, info = get_seat_blocking(models[i], model_utilities[i])
NameError: name 'get_seat_blocking' is not defined
```

Reply:
Sorry, fixed it.